accounts/templates/accounts/profile.html

```diff
@@ -0,0 +1,116 @@
+{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
+
+<link rel="stylesheet" href="{% static 'accounts/css/profile.css' %}" />
+<link rel="stylesheet" href="{% static 'accounts/css/modal.css' %}" />
+
+<div id="message_div">
+  <h1>Hi {{request.user}}!</h1>
+</div>
+<div class="content_div">
+  <div id="user_info_div">
+    <div id="profile_pic_div">
+      <img
+        src="{% static 'landing_page/images/profile.png' %}"
+        class="profile_img"
+      />
+      <div id="info">
+        <h2>{{request.user}}</h2>
+        <p>Member since: {{request.user.date_joined}}</p>
+      </div>
+    </div>
+    <div id="button_div">
+      <a href="{% url 'accounts:settings' %}">
+        <button id="settings_btn">Settings</button>
+      </a>
+    </div>
+  </div>
+
+  <div class="user_content">
+    <div class="user_lists_div">
+      <div class="list_div">
+        <h3>Recommended</h3>
+        <div class="list" id="recommended_movies">
+          <img src="{% static 'landing_page/images/darkknight.webp' %}" />
+        </div>
+      </div>
+      <div class="list_div">
+        <h3>Liked Movies</h3>
+        <div class="list" id="liked_movies">
+          <p class="placeholder_text">No movies yet!</p>
+        </div>
+      </div>
+      <div class="list_div">
+        <h3>Disliked Movies</h3>
+        <div class="list" id="disliked_movies">
+          <p class="placeholder_text">No movies yet!</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="middle_col">
+      <div class="list_div">
+        <h3>Watchlist</h3>
+        <div class="list" id="watchlist">
+          <p class="placeholder_text">No movies yet!</p>
+        </div>
+      </div>
+      <div class="list_div">
+        <h3>Watched</h3>
+        <div class="list" id="liked_movies">
+          <p class="placeholder_text">No movies yet!</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="friends_div">
+      <div class="friends_title">
+        <h2>Your Friends</h2>
+        <button class="btn btn-open">Add Friend</button>
+      </div>
+      <div class="friends_list">
```

Comment:
think it would be helpful for, when we use placeholders, to note something like
<!-- placeholders TODO create functionality -->